pkg/util/bandwidth/OWNERS

```diff
@@ -1,8 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I would be happy to hear suggestions from the community on whether the current file (package) should be deleted or retained.

@kubernetes/sig-network-approvers
@andrewsykim @aojea @bowei @caseydavenport @danwinship @dcbw @freehan @khenidak @mrhohn @robscott @thockin

Reply:
it should remain. Thank you.